src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyLeft.java

```diff
@@ -195,4 +195,21 @@ public void codeAfterLastRightCurly() {
         public Thread newThread(final Runnable r) {
             return new Thread(r);
         }}; // violation
+
+    public void testMethod() {}; //violation
```

Comment:
InputRightCurlyLeft.java is used in many tests and different config, we can not use "violation" comment as it is not always true.
Unfortunately old tests reused same inputs .... as we discovered later on it is not very maintainable approach.

Please do simple update:
move your tests code to separate Input file, that will be dedicated to exactly one test method.
please put in Input brief config in comment, good example - https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorClassDefinitionAndCommentNotSeparatedFromPackage.java

Please apply this to all other tests Inputs you changed. You can update existing Input only if it is used only in one test method.

Reply:
Done